index.html

```diff
@@ -170,30 +176,11 @@ <h1 class="project-title">Multiiii-Post-Stories Gain+Glory</h1>
             </div>
           </div>
         </div>
-
-        <div class="card">
-          <div class="data">
-            <div class="content">
-              <h1 class="project-title">Multiiii-Post-Stories Gain+Glory</h1>
-              <div class="language">
-                <ul>
-                  <li>JavaScript</li>
-                  <li>HTML</li>
-                  <li>CSS</li>
-                  <li>Ruby on rails</li>
-                </ul>
-              </div>
-              <div class="wrapper">
-                <button type="button">See Project</button>
-              </div>
-            </div>
-          </div>
-        </div>
       </div>
     </div>
   </section>
   <!-- About Myself -->
-  <section class="about-me">
+  <section class="about-me" id="about">
```

Comment:
- [ ] Your design is fantastic. In order for your design to be perfect please fix the horizontal scrolling you are presenting, it seems that one of your sections is not adapting to its container.  💪 💪 💪 
Please try adding the corresponding CSS property to stick to the design as much as possible.

Check the image below to have a better understanding.
![Capture](https://user-images.githubusercontent.com/75709904/144671096-f0f28368-530e-4d23-8ec9-be2869b1433f.PNG)